src/Language/PureScript/Linter.hs

```diff
@@ -28,8 +28,8 @@ import Language.PureScript.Types
 lint :: forall m. (MonadWriter MultipleErrors m) => Module -> m ()
 lint (Module _ _ mn ds _) = censor (addHint (ErrorInModule mn)) $ mapM_ lintDeclaration ds
   where
-  moduleNames :: S.Set Ident
-  moduleNames = S.fromList (ordNub (mapMaybe getDeclIdent ds))
```

Comment:
This seemed superfluous? Inserting into the Set is going to get rid of any duplicates anyway.

Reply:
Yeah, strange! Perhaps it was a List at one point.